contracts/contracts.go

```diff
@@ -1,59 +1,37 @@
 package contracts
 
+//go:generate go run github.com/kevinburke/go-bindata/go-bindata -prefix ../src/contracts -o assets.go -pkg contracts -nometadata -nomemcopy ../src/contracts
+
 import (
-	"fmt"
-	"io/ioutil"
-	"net/http"
 	"strings"
 
 	"github.com/onflow/flow-go-sdk"
 )
 
 const (
-	RepoBaseURL                   = "https://raw.githubusercontent.com/onflow/flow-ft/master/contracts/"
 	FungibleTokenContractFilename = "FungibleToken.cdc"
 	FlowTokenContractFilename     = "FlowToken.cdc"
+	defaultFungibleTokenAddress   = "02"
 )
 
-var (
-	fungibleTokenCode []byte
-	flowTokenTemplate []byte
-)
-
-// Load the contracts when the module is loaded.
-func init() {
-	fungibleTokenCode, _ = getContract(FungibleTokenContractFilename)
-	flowTokenTemplate, _ = getContract(FlowTokenContractFilename)
+// FungibleToken returns the FungibleToken contract interface.
+func FungibleToken() ([]byte, error) {
```

Comment:
I removed the `Contract` suffix on these functions because it seemed a bit redundant given the package name, but I can add it back!

```
import "github.com/onflow/flow-ft/contracts`

...

ft := contracts.FungibleToken()

Reply:
This is fine with me. 👍